.ghc.environment.x86_64-darwin-8.2.1

```diff
@@ -0,0 +1,60 @@
+-- This is a GHC environment file written by cabal. This means you can
```

Comment:
Please remove this file from the diff. We should probably put .ghc.environment.* to .gitignore.

Reply:
Fixed.